contracts/ethereum/packages/hatchout-cli/src/commands/ghost/list.ts

```diff
@@ -0,0 +1,39 @@
+import {Command, flags} from '@oclif/command'
+import HatchOut from "../../../../hatchout-core/hatchOut";
+
+export default class List extends Command {
+  static description = 'describe the command here';
+
+  static examples = [
```

Comment:
This is a file for `list`, but example is explaining `hello`! Is it right?

Reply:
ok